data/api/server-server/leaving-v2.yaml

```diff
@@ -109,15 +109,15 @@ paths:
                 - depth
                 - content
             example: {
-              "room_id": "!somewhere:example.org",
```

Comment:
Not sure this is accurate from the definition side - someone who knows Federarion API is needed here.

Reply:
Note that this example is not displayed in the spec. It is replaced by the example generated from the schema properties examples.

And since the schema doesn't include a `room_id` property, [the spec doesn't show one](https://spec.matrix.org/v1.7/server-server-api/#put_matrixfederationv2send_leaveroomideventid).